android/app/src/debug/java/com/hathormobile/ReactNativeFlipper.java

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Flipper looks great!

But, did this come from the diff, or did you add it manually?

Reply:
It came from the diff, I kept it since it appeared to be very useful.